### PR TITLE
first_post

### PR DESCRIPTION
**Pull request by manasvini2906**
…t-manasvini2906.md

### CODE REVIEW

**Review by shashwatj14 — CHANGES_REQUESTED** [review]
To keep things simple, change the author name in the title from manasvini2906 to just Manasvini.

File: _posts/manasvini2906/2022-09-22-First-post-manasvini2906.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
-layout = post
-author = manasvini2906
-title = 'manasvini2906's first post!'
+layout : post
+author : manasvini2906
+title : 'manasvini2906's first post!'
```
Comment (shashwatj14) [inline _posts/manasvini2906/2022-09-22-First-post-manasvini2906.md:4]: title: 'Manasvini's first post!'

**Review by atUtexas — APPROVED** [review]
Making this change will definitely help your post appear in the people's section of our class website. If it still doesn't work, consider using parenthesis"" for the title.

**Comment by atUtexas** [comment]
Making this change will definitely help your post appear in the people's section of our class website. If it still doesn't work, consider using parenthesis"" for the title.

**Pull request merged**